simulator/simulator-view/src/shared/components/main-map/main-map.component.ts

```diff
@@ -26,87 +18,37 @@ import { MatButtonModule } from '@angular/material/button';
 export class MainMapComponent implements OnInit, AfterViewInit {
   map!: L.Map;
   isInMenuCreationMode: boolean = false;
-  $intensitySubscription: Subscription = new Subscription();
   $isInCreationSubscription: Subscription = new Subscription();
 
   private defaultZoomLevel = 20;
   constructor(
     private fireCreationService: FireCreationService,
-    private fireMarkerService: FireMarkerService,
+    private fireStationMarkerService: FirestationMarkerService,
     private interventionMarkerService: InterventionMarkerService,
-    private fireStationMarkerService: FirestationMarkerService
+    private fireMarkerService: FireMarkerService
   ) {}
 
   ngOnInit(): void {
     this.isInMenuCreationMode = this.fireCreationService.isSettingNewElement;
     this.$isInCreationSubscription =
       this.fireCreationService.$isInCreationState.subscribe((isCreating) => {
         this.isInMenuCreationMode = isCreating;
-        if (isCreating) {
+        if (!isCreating) {
+          this.fireMarkerService.fetchAll(this.map);
```

Comment:
Dans ce cas peut etre ajouter un state de is done non ? car sinon ca va se lancer dés que la page aura charger une premiere fois alors que le mode création n'aura meme pas été lancé une seule fois